iModelJsNodeAddon/JsCloudSqlite.cpp

```diff
@@ -191,8 +191,20 @@ struct JsCloudContainer : CloudContainer, Napi::ObjectWrap<JsCloudContainer> {
 
     void RequireWriteLock() {
         RequireConnected();
-        if (!m_writeLockHeld)
-            BeNapi::ThrowJsException(Env(), Utf8PrintfString("container [%s] is not locked for write access", m_containerId.c_str()).c_str());
+        BeJsDocument lockedBy;

```

Comment:
Does the requireWriteLock function still throw an error 'the container is not locked for write access'? If the user never calls acquireWriteLock?

Reply:
To me, it seems like the answer is no. We should have a test on itwinjs side that asserts we get that error if we never held the write lock in the first place.